src/DIRAC/ConfigurationSystem/Client/Utilities.py

```diff
@@ -578,3 +569,41 @@ def getDIRACGOCDictionary():
 
   log.debug('End function.')
   return S_OK(dictionary)
+
+
+def getAuthAPI():
+  """ Get Auth REST API url
+
+      :return: str
+  """
+  return gConfig.getValue("/Systems/Framework/%s/URLs/AuthAPI" % getSystemInstance("Framework"))
+
+
+def getAuthorizationServerMetadata(issuer=None):
+  """ Get authorization server metadata
+
+      :return: S_OK(dict)/S_ERROR()
+  """
+  result = gConfig.getOptionsDictRecursively('/DIRAC/Authorization')
```

Comment:
I would say that this dict should be found in the *Operations* section of the CS, not in a new section.

Reply:
this should be defined on the DIRAC client side as well

Reply:
I see, it's basically for bootstrap. But a server can support multiple VOs with different IdPs and AS for each VO, that's why server side the *Operations* section would be better.

Reply:
I would suggest this change in order not to introduce a new section
```suggestion
  result = gConfig.getOptionsDictRecursively('/DIRAC/Security')
```

Reply:
@fstagni : As far as I understand (Andrii can give more details), the DIRAC Authorization Server is a single service for all the served VO's of the installation which acts as a proxy to all the VO-specific IdP services. There will be no DIRAC AS per VO. So, describing it in the central, e.g. /DIRAC/Security section, is quite appropriate

Reply:
`Authorization` section moved to `/DIRAC/Security` section.